### DISCUSSION TITLE
How many eight-thousanders are there in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#478] [Galaxy Trivia] [answered]
**Bonus:** Who was the first person to climb all of them?

**Comment by peterbaccaro** [comment, marked as answer]
The eight-thousanders are the **14** independent mountains on Earth that are more than 8,000 metres (26,247 ft) high above sea level.
On 16 October 1986, Italian **Reinhold Messner** became the first person to climb all 14 eight-thousanders.
